backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/view/ViewPropertiesConfigurer.java

```diff
@@ -278,4 +296,34 @@ private SelectDescription createShapeSelectionField() {
         };
     }
 
+    private Function<VariableManager, List<Object>> getDiagnosticsProviders(String featureName) {
+        return variableManager -> {
+            var optionalSelf = variableManager.get(VariableManager.SELF, EObject.class);
+
+            if (optionalSelf.isPresent()) {
+                EObject self = optionalSelf.get();
+                List<Object> validate = this.validationService.validate(self, featureName);
+                return validate;
+            }
+
+            return List.of();
+        };
+    }
+
+    private String kindProvider(Object object) {
+        if (object instanceof Diagnostic) {
+            Diagnostic diagnostic = (Diagnostic) object;
+            return diagnostic.getKind();
+        }
+        return ""; //$NON-NLS-1$
+    }
+
+    private String messageProvider(Object object) {
+        if (object instanceof Diagnostic) {
+            Diagnostic diagnostic = (Diagnostic) object;
+            return diagnostic.getMessage();
+        }
+        return ""; //$NON-NLS-1$
+    }
```

Comment:
In both of those situations, the diagnostic could stay the EMF diagnostic without any issue.

Reply:
This PR should probably start by this change for the validation side

Reply:
Done